bed/methods.go

```diff
@@ -9,18 +9,23 @@ import (
 // Current methods satisfy requirements for the following interfaces:
 // bed.BedLike
 
+// GetChrom grabs the chrom of the bed struct
 func (b Bed) GetChrom() string {
 	return b.Chrom
 }
 
+// GetChromStart grabs the starting coordinates of the bed struct
 func (b Bed) GetChromStart() int {
 	return b.ChromStart
 }
 
+// GetChromEnd grabs the end coordinates of the bed struct
 func (b Bed) GetChromEnd() int {
 	return b.ChromEnd
 }
 
+// UpdateCoord will change the bed struct to the user defined chrom string,
```

Comment:
I believe changing/modifying the bed struct was the original plan, but this function right now creates a copy of the bed struct and returns that copy with modified coordinates (and the original is unchanged).